charts/portal/templates/tenant-provisioner/tenant-provisioner-deployment.yaml

```diff
@@ -70,7 +70,7 @@ spec:
             - containerPort: 9001
               protocol: TCP
           env:
-            - name: RABBITMQ_PASSWORD
+            - name: RABBITMQ_PASS
```

Comment:
Where is RABBITMQ_PASS coming from?

Reply:
this is creating it with the value set in values yaml